core/src/main/scala/kafka/server/ForwardingManager.scala

```diff
@@ -161,9 +178,13 @@ class ForwardingManagerImpl(
       }
     }
 
+    forwardingManagerMetrics.queueLength.getAndIncrement()
     channelManager.sendRequest(envelopeRequest, new ForwardingResponseHandler)
   }
 
+  override def close(): Unit =
```

Comment:
This is a minor point, but I don't think it makes sense for the forwarding manager to close the metrics here. That metrics object is used by other components, and we already have code to close it elsewhere, right?

Reply:
This only closes the metrics and sensors associated with `ForwardingManagerMetrics`, which is only used by `ForwardingManager`.

Reply:
ok